### ISSUE TITLE
Unable to Dismiss Notifications Pane If No Tabs are Open and "Always show panel tabs" is Unchecked

### ISSUE THREAD

**Issue by jcefoli** [mRemoteNG/mRemoteNG#368]
It's not possible to close the notification pane and the UI looks strange if it is the only "tab" open and the "Always show panel tabs" option is unchecked.

**To reproduce:**
1. Verify setting: Tools --> Options --> Tabs & Panels --> Uncheck "Always show panel tabs"
2. Make sure there are no connections/external apps/port scan etc enabled or connected
3. Enable Notification Pane (View --> Notifications)

**Actual**
UI glitch shows the notification pane, taking up valuable screen real estate with no (obvious) way to close it.
![image](https://cloud.githubusercontent.com/assets/6618013/22301392/ee42e2a0-e2f8-11e6-8689-ead9c4d7fff9.png)
(reproducible in the latest develop branch, probably always was an issue, probably OS agnostic)

**Expected**
1) There should be a way to dismiss the notifications pane if "Always show panel tabs" is unchecked and it's the only (hidden) tab opened

**Workarounds**
1. Tools --> Options --> Tabs & Panels --> Check "Always show panel tabs", and then you're able to X out the notification pane, as the tab will be visible... OR
2. If you have the above setting unchecked, connect to a few resources that open multiple items on the tab panel, and the notifications tab will show.. OR
3. Disable the notification pane in View --> Notification Pane

You mentioned this code is a little messy in chat. If there's a quick interim fix, let's do that, but I fully understand if it will take some time for a proper rewrite/redesign and isn't a priority, since we understand what's going on now and how to fix it.

We can add this to the known issues wiki in case anyone gets confused, and there are workarounds

**Comment by sparerd** [comment]
I'm not even sure why we have an option for `Always show panel tabs`, tbh. Having it not checked leads to poor UX in some cases (like this one). My vote it to remove the `Always show panel tabs` option and have it always show the tabs. At the very least, it should be defaulted to checked (to show the tabs).

**Comment by kmscode** [comment]
The problem with Always show panel tabs, is when you might have just a single panel tab (General by default), it's just a waste of space. I think this needs to be re-thought after #155 and #269 

Unfortunately, it looks like newer versions of DockPanelSuite (#269) seem to also waste some space, but this might be mitigated by getting rid of MagicLibrary (#155)....

**Comment by kmscode** [comment]
@jcefoli What do you think about this one in the latest removemagiclib build? Solved or still need some TLC?

**Comment by jcefoli** [comment]
It is a lot better now that it defaults to the bottom. The contents of the pane itself (when expanded) could use a redesign, but that'd be another ticket. I think you can close this, as it's no longer relevant (since the new default location of the notification pane as part of RemoveMagicLib is much better).

**Comment by kmscode** [comment]
closing this one per previous comments. @jcefoli I'm sure you'll open a new issue if you can think of any specific improvements in this area... :wink: Thanks!!!
